tests/test_solution_variables.py

```diff
@@ -15,14 +15,14 @@ def test_solution_variables(new_solver_session):
     import_file_name = examples.download_file(
         "mixing_elbow.cas.h5", "pyfluent/mixing_elbow"
     )
+    examples.download_file("mixing_elbow.dat.h5", "pyfluent/mixing_elbow")
 
     solution_variable_info = solver.fields.solution_variable_info
     solution_variable_data = solver.fields.solution_variable_data
+    assert solution_variable_info == solver.svar_info
+    assert solution_variable_data == solver.svar_data
 
-    solver.file.read(file_type="case", file_name=import_file_name)
-
-    solver.solution.initialization.hybrid_initialize()
-    solver.solution.run_calculation.iterate(iter_count=10)
```

Comment:
Probably already considered, but are there other tests that still run some calculations? I don't think all tests need to run calculations, but certainly at least some of them should still do that, to cover this scenario

Reply:
Some tests like `test_field_data` compare results, so specified iterations are necessary (we can think of some alternate strategy like results change than results comparison, however the test is relative fast as of now). Some nightly/weekly tests runs iterations, I'm optimizing the PR tests only for now.